### DISCUSSION TITLE
Does upgrading Plex's "Library" kill the oldish SageTV agent functionallity?

### DISCUSSION THREAD

**Discussion by eelpout** [OpenSageTV/Community#175] [General]
Plex server keeps nagging me to upgrade my library, but I'm afraid if I do that it'll make the SageTV agent inoperable. Anyone know for sure? can these agents be easily upgraded if necessary?

**Comment by mgpaulus** [comment]
I breaks it in that the sagetv specific scanner won't work any longer, and therefore your agent won't get the proper information to effectively pull the proper metadata.  So, you will see your programs/shows/movies in Plex, but you will see them as the recorded name (ie:  The 5-Year Christmas Party - Directed by Peter Benson - 119724299-0), and that's it, no description, etc.

Supposedly, sometime next year, plex will be releasing a way to either import information via a custom .nfo file, and/or several apis to be able to create custom components (again).  Will just have to wait and see. 

If your stuff is still working, I would suggest leaving it until they get the new apis and new import process(es) in place.

**Reply by eelpout** [reply]
yeah, it's still working so I will resist the temptation to click on the "exclamation point!!!!!" on the server and upgrade. ;) 

Thanks!
